index.html

```diff
@@ -0,0 +1,223 @@
+        <p>I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, 
+          don<span>&#39;</span>t hestiate to contact me.</p>
+      </div>
+      <div class="icons-container">
+        <ul>
+          <li><a><img src="assets/github.png" alt="github"/></a></li>
+          <li><a><img src="assets/linkedin.png" alt="linkedin"/></a></li>
+          <li><a><img src="assets/angel list.png" alt="angel-list"/></a></li>
+          <li><a><img src="assets/twitter.png" alt="twitter"/></a></li>
+          <li><a><img src="assets/medium.png" alt="medium"/></a></li>
+        </ul>
+        </div>
+    </section>  
+      <!-- Recent-Works -->
+    <section class="recent-works">
+
+        <h2>My Recent Work</h2>
+      <hr>  
+      <div class="head-work">
+        <img src="assets/Img Placeholder.png" alt="project-cover">
+        <h2>Multi-Post Stories</h2>
+        <p>A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a standard dummy text.</p>
+        <ul class="techstack-container">
+          <li>css</li>
+          <li>html</li>
+          <li>bootstrap</li>
+          <li>Ruby</li>
+        </ul>
+        <a class="project-button">See Project</a>
+      </div>
+    </section>
+
+    <section class="works">
+      <div class="work">
+        <h2>Profesional Art Printing Data</h2>
+        <p>A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a standard dummy text.</p>
+        <ul class="techstack-container">
+          <li>html</li>
+          <li>bootstrap</li>
+          <li>Ruby</li>
+        </ul>
+        <button class="project-button">See Project</button>
+      </div>
+
+      <div class="work">
+        <h2>Profesional Art Printing Data</h2>
+        <p>A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a standard dummy text.</p>
+        <ul class="techstack-container">
+          <li>html</li>
+          <li>bootstrap</li>
+          <li>Ruby</li>
+        </ul>
+        <button class="project-button">See Project</button>
+      </div>
+
+      <div class="work">
+        <h2>Profesional Art Printing Data</h2>
+        <p>A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a standard dummy text.</p>
+        <ul class="techstack-container">
+          <li>html</li>
+          <li>bootstrap</li>
+          <li>Ruby</li>
+        </ul>
+        <button class="project-button">See Project</button>
+      </div>
```

Comment:
Please consider wrapping the **_div tag_** inside an **article tag** to make it semantically correct 👍